static/js/import_wizard2/templates.tsx

```diff
@@ -197,7 +198,7 @@ export const TEMPLATE_OPTIONS: { [templateId: string]: TemplateInfo } = {
 
 export interface MappingTemplateProps {
   csvData: CsvData;
-  predictedMapping: Mapping;
+  userMapping: Mapping;
```

Comment:
I feel like userMapping may be more accurate here because this will be updated with the latest userMapping? every time user makes an update, this object will be updated

Reply:
ok, sg